.github/workflows/artifact-workflow.yml

```
@@ -0,0 +1,51 @@
+name: PR artifacts
+on:
+  workflow_run:
+    workflows: [🐧 Linux Builds,🖥️ Windows Builds]
+    types: [completed]
```

You don't brackets for one item.
```suggestion
    types: completed
```